.github/workflows/merge.yml

```
@@ -0,0 +1,38 @@
+name: Merge
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)